var/www/so/capme/.inc/callback-elastic.php

```diff
@@ -308,8 +330,8 @@ function cliscript($cmd, $pwd) {
 				}
 
 				// destination_ip
-				if (isset($elastic_response_object["hits"]["hits"][0]["_source"]["destination_ip"])) {
-					$dip = $elastic_response_object["hits"]["hits"][0]["_source"]["destination_ip"];
+				if (isset($elastic_response_object["hits"]["hits"][$key]["_source"]["destination_ip"])) {
+					$dip = $elastic_response_object["hits"]["hits"][$key]["_source"]["destination_ip"];
 					error_log($dip);
```

Comment:
Let's go ahead and remove these `error_log()` lines.

Reply:
Will do.